.eslintrc.json

```diff
@@ -2,11 +2,25 @@
   "parser": "@babel/eslint-parser",
   "plugins": [
     "react",
-    "no-jquery"
+    "no-jquery",
+    "jsdoc"
```

Comment:
@driusan jsdoc has been added to the plugins. When  `"valid-jsdoc": "off"` is set to "error" then we will get errors when running `npm run compile` such as if comments are missing `@return` or correct `@param` value. I believe @laemtl has it set to off currently because she's cleaning up the comments in parts by separate PRs.

Reply:
thanks

Reply:
Oups I forgot to add a description. Thanks @maltheism for the clarifications!